app/models/site.rb

```diff
@@ -1,5 +1,6 @@
 class Site < ActiveRecord::Base
   belongs_to :user
+  has_many :reviews
```

Comment:
what happens to reviews if a site gets deleted? Be sure to check out destroy dependencies on the associations article of the rails guides